esx_service/cli/vmdkops_admin.py

```diff
@@ -1006,15 +1006,46 @@ def tenant_access_add(args):
         error_info = "Tenant {0} does not exist".format(args.name)
         return operation_fail(error_info)
 
+    privileges = [d for d in tenant.privileges if d[auth_data_const.COL_DATASTORE] == args.datastore]
```

Comment:
nit: I think this construct used in many places.... maybe its worth to wrap in a function